src/app/Peerly/screens/GiveAppreciationScreen/index.tsx

```diff
@@ -159,12 +160,17 @@ const AppreciationScreen = () => {
             <Controller
               control={control}
               render={({field: {onChange, value}}) => (
-                <TextInput
-                  style={styles.description}
-                  onChangeText={onChange}
-                  value={value}
-                  multiline
-                />
+                <>
+                  <TextInput
+                    style={styles.description}
+                    onChangeText={onChange}
+                    value={value}
+                    multiline
+                  />
+                  <Typography style={styles.noteText}>
```

Comment:
The text field should toggle between helper text and error messages. We should avoid showing both simultaneously, especially when they have same text.